src/frontend/src/planner/mod.rs

```diff
@@ -27,6 +27,7 @@ mod set_expr;
 mod statement;
 mod update;
 mod values;
+pub use query::LIMIT_ALL_COUNT;
```

Comment:
Looks a little weird to me. Can we refactor offset into `Option` in the future?

Reply:
Yes, this will need some changes in optimizer.